src/lib.rs

```diff
@@ -62,8 +62,8 @@ fn supports_color(stream: Stream) -> usize {
     } else if std::env::var("COLORTERM").is_ok()
         || std::env::var("TERM").map(|term| check_ansi_color(&term)) == Ok(true)
         || std::env::consts::OS == "windows"
+        || std::env::var("CLICOLOR").map_or(false, |v| v != "0")
         || is_ci::uncached()
-        || std::env::var("CLICOLOR") != Ok("0".into())
```

Comment:
why make this change?

Reply:
I think it makes sense if CI checks should be the last one.

Also, I just noticed that inequality check is probably not a good way to check results. `map_or` should be preferred.